script.js

```diff
@@ -1,36 +1,17 @@
 function drawChess() {
-    let mainBlock = document.querySelector('.main-board');
-
-    let block;
-
-    let flag = true;
-
+    const mainBlock = document.querySelector('.main-board');
+    
     for (let i = 0; i < 8; i++) {
         for (let j = 0; j < 8; j++) {
-
-            if (j == 0) {
-                flag = !flag;
-            }
-
-            block = document.createElement('div');
-
-
-            if (flag) {
-                block.className = 'block black';
+            const block = document.createElement('div');
+            mainBlock.append(block);

```

Comment:
Can you add this block to the parent after manipulation with it? (after changes of classes)